### DISCUSSION TITLE
Virustotal results 4 / 68

### DISCUSSION THREAD

**Discussion by MojtabaFaal** [DNSCrypt/dnscrypt-proxy#1823] [Q&A] [answered]
Hi
I searched the file on **Virustotal** website and it says that the file has a virus?
[https://www.virustotal.com/gui/file/b1a3091b7380768f1f3b1e6e964e29f94263a3c2f068e9c8b7de1dad43db8a52/detection](https://www.virustotal.com/gui/file/b1a3091b7380768f1f3b1e6e964e29f94263a3c2f068e9c8b7de1dad43db8a52/detection)
Seriously?
Why ?
What is the problem ?

**Comment by Kaiyangshi-Ito** [comment]
Same question.

**Comment by jedisct1** [comment, marked as answer]
If you downloaded it from here (and you can also check the signature with `minisign`), there is nothing to fear about.

AV products just assign a score to files according to heuristics, and frequently report false positives.

Now you know 4 of them that may be not worth using :)

The way to fix these is to report false positives, so that they can improve their algorithms. So, go to their website and try to find how to do it!

**Reply by MojtabaFaal** [reply]
Thanks
